text/1071-cell.md

```diff
@@ -84,7 +84,7 @@ interface Cell<Value> extends Reactive<Value> {
     /**
     * Utility to create a Cell without the caller using the `new` keyword.
     */
-    static create<T>(initialValue: T, equals?: (a: T, b: T) => boolean): Cell<T>;
```

Comment:
should we support `lazy` values?
For example, if initial value is a tricky logic, there is no reason to resolve it during cell creation, because it may be used later.

```ts
// a-lot of compute
const value = new Array(10000).fill(null);
const arrays = Cell.create(value);

// lazy, no compute 
const value = () => new Array(10000).fill(null);
const arrays = Cell.createLazy(value);
```

Reply:
What would the implementation of createLazy potentially look like?

Reply:
Like this: https://github.com/lifeart/glimmer-next/blob/master/src/utils/reactive.ts#L142 it's not 100% accurate, but main idea is same

Reply:
Don't dynamically created/modified objects create extra memory pressure?

Reply:
@NullVoxPopuli it is, but it depends on usecase, in browser we limited by CPU, not by memory :)

Reply:
I don't see why it couldn't be added later as well

It may make more sense to do lazy construction in objects/arrays/maps, etc?